src/core/public/chrome/chrome_service.tsx

```diff
@@ -154,7 +154,7 @@ export class ChromeService {
   }: StartDeps): Promise<InternalChromeStart> {
     this.initVisibility(application);
 
-    const appTitle$ = new BehaviorSubject<string>('OpenSearchDashboards');
+    const appTitle$ = new BehaviorSubject<string>('OpenSearch Dashboards');
```

Comment:
should we change `OpenSearchDashboards` to `Overview`? since issue [428](https://github.com/opensearch-project/OpenSearch-Dashboards/issues/428) you solved in this PR requests 'The header for Overview pages says "OpenSearchDashboards" (no space). It should say "Overview" '